packages/web-frontend/src/components/View/NoDataMessage.js

```diff
@@ -22,7 +22,7 @@ const NoDataMessageText = styled.p`
 export const NoDataMessage = ({ viewContent }) => {
   let message = 'No data';
 
-  if (viewContent.noDataMessage) {
+  if ('noDataMessage' in viewContent) {
```

Comment:
This will not work if `noDataMessage` is explicitly set to `null`/`undefined`: 
```js
const viewContent= { noDataMessage: undefined };
console.log('noDataMessage' in viewContent); // true
```

But it's probably fine for our purposes here.